src/action/common/configure_init_service.rs

```diff
@@ -471,6 +458,28 @@ impl Action for ConfigureInitService {
                 )
                 .await
                 .map_err(Self::error)?;
+
+                for attempt in 1..100 {
```

Comment:
once the logic issue in the code below this point is fixed, this comment will be accurate

```suggestion
                // check if the daemon is down up to 99 times, with 100ms of delay between each attempt
                for attempt in 1..100 {
```